### PR TITLE
Add 3 dotnet tools

### PR DESCRIPTION
**Pull request by awaescher**
Add Fusion++, RepoZ and dnSpy to the Tools list.

### CODE REVIEW

File: dotnet-developer-projects.md [hunk 1]
```diff
@@ -249,12 +249,14 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [CInject](https://github.com/punitganshani/CodeInject) - A tool to inject your C#/VB.NET code into existing .NET assemblies and executables
   * [CppSharp](https://github.com/mono/CppSharp) - C/C++ bindings generation for .NET.
   * [Crawler-Lib Build Tools](http://download.crawler-lib.net/BuildTools/) -  [A PowerShell based toolbox](http://www.crawler-lib.net/build-tools) for automating the software build process, autmated testing and integration.
+  * [dnSpy](https://github.com/0xd4d/dnSpy) - .NET decompiler, debugger and assembly editor
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]:
```suggestion
  * [dnSpy](https://github.com/0xd4d/dnSpy) - A .NET decompiler, debugger, and assembly editor that lets you edit and debug assemblies even if you don't have their source code.
```

File: dotnet-developer-projects.md [hunk 2]
```diff
@@ -249,12 +249,14 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
...
   * [FAKE](http://fsharp.github.io/FAKE/) - Build automation system with capabilities which are similar to make and rake.
   * [FlubuCore](https://github.com/flubu-core/flubu.core) - A cross platform build and deployment automation system for building projects and executing deployment scripts using C# code.
   * [Fody](https://github.com/Fody/Fody) - Extensible tool for weaving .NET assemblies.
   * [FsEye](https://github.com/swensensoftware/fseye) - A visual object tree inspector for the F# Interactive.
   * [FsharpLint](https://github.com/fsprojects/FSharpLint) - Lint tool for F#.
   * [FsPickler](https://github.com/nessos/FsPickler) - Serialization library that facilitates the distribution of .NET objects.
+  * [Fusion++](https://github.com/awaescher/Fusion) - A modern alternative to the Microsoft Assembly Binding Log Viewer (FUSLOGVW.exe)
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]:
```suggestion
  * [Fusion++](https://github.com/awaescher/Fusion) - A modern alternative to the Microsoft Assembly Binding Log Viewer (FUSLOGVW.exe).
```

File: dotnet-developer-projects.md [hunk 3]
```diff
@@ -269,6 +271,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [Polly](https://github.com/App-vNext/Polly) - A resilience and transient-fault-handling library that allows developers to express policies such as Retry, Circuit Breaker, Timeout, Bulkhead Isolation, and Fallback in a fluent and thread-safe manner.
   * [Protobuf-net](https://github.com/mgravell/protobuf-net) - A .NET implementation of protobuf, allowing you to serialize your .NET objects efficiently and easily.
   * [Quick Diagram Tool for C#](https://github.com/realvizu/QuickDiagram) - Code visualization tool for C#.
+  * [RepoZ](https://github.com/awaescher/RepoZ) - A zero-conf git repository hub for Windows and macOS with Windows Explorer- & CLI-enhancements
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]:
```suggestion
  * [RepoZ](https://github.com/awaescher/RepoZ) - A zero-conf git repository hub for Windows and macOS with Windows Explorer and CLI enhancements.
```

**Comment by awaescher** [comment]
Thanks for the changes, @rpetrusha. I changed applied them, of course.

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @awaescher. I'll merge your PR now.

**Pull request merged**